tests/test_model.py

```diff
@@ -0,0 +1,636 @@
+"""Test for Model class"""
+from datetime import datetime
+
+from custom_components.teslafi.const import TESLAFI_DATE_FORMAT
+from custom_components.teslafi.model import (
+    TeslaFiVehicle,
+    TeslaFiTirePressure,
+)
+
+def test_TeslaFiVehicle_constructor():
+    vehicle = TeslaFiVehicle({})
+    
+    assert isinstance(vehicle, TeslaFiVehicle)
+    
+def test_TeslaFiVehicle_update_non_empty():
+    """Test TeslaFiVehicle class"""
+    vehicle_data = {
+        "vin": "5YJSA1CG3DFP14555",
```

Comment:
small note: let's just make sure we're not inadvertently using someone's "real" VIN. E.g., use something that's obviously intended as a test value for the serial number portion: `…00000`, `99999`, `12345`.